boxoffice/views/order.py

```diff
@@ -438,6 +439,7 @@ def payment(order):
                 invoice_no=order.invoice_no,
             ),
         )
+        send_telegram_message.queue(order=order)
```

Comment:
Never pass an object to an RQ job. It uses pickles to pass data and that's a security risk. Pass a simple data type like an int. See the previous line for an example.